platform/linux-generic/odp_packet.c

```diff
@@ -1612,8 +1612,9 @@ void odp_packet_print(odp_packet_t pkt)
 		int str_len;
 
 		str_len = snprintf(&seg_str[0], max_len,
-				   "    seg_len    %-4" PRIu32 "  seg_data %p "
-				   "ref_cnt %u\n",
+				   "  seg_len      %-4" PRIu32 "\n"
+				   "  seg_data     %p\n"
+				   "  ref_cnt      %u\n",
```

Comment:
The reason for the original format was to make printing per segment data clearer.

E.g.
```
Packet
------
  pool index   1
  buf index    118
  ev subtype   1
  input_flags  0xf3fdff0
               l2 l3 l4 eth vlan arp ipv4 ipv6 ipsec udp tcp sctp icmp 
  flags        0x0
  cls_mark     0
  l2_offset    2
  l3_offset    16
  l4_offset    30
  frame_len    1536
  input        0
  headroom     128
  tailroom     1152
  num_segs     2
    seg_len    512   seg_data 0x7ff28f058b00 ref_cnt 0
    seg_len    1024  seg_data 0x7ff28f0580c0 ref_cnt 0
```

To clarify this further, perhaps the segment index could be printed as well.

```
...
  headroom     128
  tailroom     1152
  num_segs     2
    [0] seg_len  512   seg_data 0x7ff28f058b00 ref_cnt 0
    [1] seg_len  1024  seg_data 0x7ff28f0580c0 ref_cnt 0
```

Reply:
Thanks for your reply, @matiaselo . I understand the intent of the original format now. I will make the changes to print the segment index instead.